pipelines/example/measurement_visit_table_selectors_example.yaml

```diff
@@ -0,0 +1,64 @@
+description: Compute per-visit metrics from sourceTable_visit catalogs
+tasks:
+  nsrcMeasVisitTable:
+    class: lsst.faro.measurement.VisitTableMeasurementTask
+    config:
+      connections.package: info
+      connections.metric: nsrcMeasVisitTable
+      measure.selectorActions.FlagSelector.selectWhenTrue: ["detect_isPrimary"]
```

Comment:
Don't forget to either (a) move this back down to the python block with the other config options, or (b) move all the configs up to this block.

Reply:
moved down.